CDCgov/prime-reportstream
# Flexion org settings to handle results

This PR adds sender and receiver settings for the TI results (ORU_R01) use case as described in [TI's issue #640](https://github.com/CDCgov/trusted-intermediary/issues/640)

Test Steps (local):
1. Run `./prime multiple-settings set --input ./settings/staging/0166-flexion-staging-results-handling.yml`
2. Send an ORU message to the waters api with the `flexion.simulated-lab` client (use [004_MN_ORU_R01_NBS.hl7](https://github.com/CDCgov/trusted-intermediary/blob/main/examples/MN/004_MN_ORU_R01_NBS.hl7) if you need sample ORU)
    `curl --header 'Content-Type: application/hl7-v2' --header 'Client: flexion.simulated-lab' --header 'Authorization: Bearer token' --data-binary '@ORU_R01_MN_natus.hl7' 'http://localhost:7071/api/waters'`
4. Verify the request went through without any errors and a FHIR file was uploaded to the sftp folder

### Extras

Who will do the push to staging? I think TI folks can do it but we would like to verify if the RS team wants to handle that instead. But I think some TI engineers are able to use the CLI and push the settings changes (only to staging).

## Checklist

### Testing
- [x] Ran `./gradlew testSmoke`
- [x] Ran `./gradlew testIntegration`
